Week3/rotated array.c

```diff
@@ -0,0 +1,103 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+ void main()
+  {
+  	int arr[10],s,search,a=0,first,last,middle,i,b=0,c=0;
+  	
+  	//array input 
+  	
+  	   printf("size of array = \n");
+  	   scanf("%d",&s);
+  	   printf("enter the elements of the array -> \n");
+  	   
+  	for(i=0;i<s;i++)
+  	 {
+  	 	scanf("%d",&arr[i]);
+  	 	
+	 }
+  	    
+  	    printf("enter the number to find -> ");
+	    scanf("%d",&search);	   
+  	    
+  	   
+  	for(i=0;i<s;i++)    
+	  {

```

Comment:
This is already o(n) if I have to scan the array then what is the use of your algorithm ? The question specifically asks to do it in log n